tests/e2e/targetallocator-features/00-assert.yaml

```diff
@@ -56,6 +56,8 @@ spec:
                     operator: In
                     values:
                       - "true"
+      securityContext:
+        runAsUser: 1000
```

Comment:
should we assert the other properties here too?
```
      runAsGroup: 3000
      fsGroup: 2000
```

Reply:
added